### PR TITLE
Completed the css-positioning exercise

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 14.5rem;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:16]: Please revise this to use smaller measurements from `right` as we discussed in Q&A

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 9.5rem;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:29]: Same here, use `bottom`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! Please see my notes

**Comment by owen-adams2021** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**